### PR TITLE
New Cheat Sheet of Magic of Numbers for Number System

### PR DESCRIPTION
**Pull request by Harsh061**
**What does your Instant Answer do?**
My Instant answer provide short cut tricks for calculations.

**What is the data source for your Instant Answer? (Provide a link if possible)**
Word Pandit (http://wordpandit.com/magic-of-numbers/)

**Why did you choose this data source?**
Because it helps us to prepare for competitive exams and provide shortcut tricks for calculations.

**Are there any other alternative (better) data sources?**
Might Be

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Students preparing for competitive exams and lovers of aptitude.

**Where did you hear about DuckDuckHack? (For first time contributors)**
Navneet Sharma (Contributor).

**Screen Shots of Instant Answer**
magic-number

#DDG JAIPUR

---
https://duck.co/ia/view/magic_number_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@Harsh061 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by MariagraziaAlastra** [comment]
Hey @Harsh061, thanks for your contribution!
Did you know you don't have to fill the PR template anymore?
You just need to create an IA Page [here](https://duck.co/ia/new_ia) and then paste the link to it in your first comment! 
Read more about the process for submitting a PR [in the docs](http://docs.duckduckhack.com/submitting/pull-request.html#submitting-a-pull-request).
Thanks!

**Comment by navneetzz** [comment]
https://duck.co/ia/view/magic_number_cheat_sheet

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/magic_number_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/magic_number_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@Harsh061 this is looking great! It looks like you haven't filled out the info on your IA page though. Please be sure to fill out a brief description, a source and some example queries that you know trigger the cheat sheet

File: share/goodie/cheat_sheets/json/magic-number.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{
+    "id": "magic_number_cheat_sheet",
+    "name": "Magic of Number",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/magic-number.json]: I think you're missing an `"s"` here? Magic of Numbers?

**Comment by moollaza** [comment]
@Harsh061 this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

If you need ideas about how to contribute next, please feel free to reach out to me on Slack! I'm happy to chat about how you can help :+1:

**Comment by tagawa** [comment]
@Harsh061, thanks again for creating the Magic of Numbers cheat sheet! You’ve done a really good job here and we'd like you to help make DuckDuckGo even better for yourself and others interested in maths.

When you're ready to move on, please:

- Create a new Instant Answer for searches you do related to maths

OR

- Test live maths Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Creating-or-Improving-Instant-Answers).

Thanks again!

**Pull request merged**